{"name": "test_multiple_login_page_success[invalid_user2-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user2'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753593329763, "stop": 1753593332610, "uuid": "18de6892-96fb-4b2a-ba89-6cf9bbbe5f07", "historyId": "bff793983932d93056520584ccf3f372", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "19652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}